curacmd/install

```diff
@@ -1 +1,150 @@
-# Curacmd Bootstrap / Install Script
+#!/bin/bash
```

Comment:
Nit: How do you feel about using `.sh` as the extension? I think it helps identify the type of file, but I'm not tied to it.